playbooks/prepare_host.yaml

```diff
@@ -48,6 +48,25 @@
           owner: stack
           group: stack
 
+  # To ensure that our CentOS users are actually using Stream.
+  # TripleO Master only support Stream now.
+  - name: Migrate to CentOS Stream if not done already
```

Comment:
We should probably reboot the node for the distro change to fully take effect.

The command is also printing a warning, we should consider setting `warn: false` as the dnf module apparently [doesn't support the two commands used here](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/dnf_module.html).

```
[WARNING]: Consider using the dnf module rather than running 'dnf'.  If you need to use command because dnf is insufficient you can add 'warn: false' to this command task or set 'command_warnings=False' in ansible.cfg to get rid of this message.
```